test/node/plugins.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+var $nocca = require('../../index');
+
+var Nocca = new $nocca({
+    endpoints: {
+        'google': {
+            targetBaseUrl: 'http://www.google.com/'
+        },
+        'yahoo': {
+            targetBaseUrl: 'http://www.yahoo.com/',
+            keyGenerator: 'keyGeneratorFactory',
+            keyGeneratorFactoryMethod: 'urlAndHeadersKeyGeneratorBuilder'
+        },
+        '_default': {
+            targetBaseUrl: 'http://localhost:3004/'
+        }
+    },
+    keyGenerator: 'cherryPickingKeygen',
```

Comment:
in this example cherryPickingKeygen is used as default but is overridden in the yahoo endpoint (notice the keyGeneratorFactoryMethod property with the function name as value).

There's no regex functionality in the cherry picker, not sure if other features from the factory are not present in the cherry picker.